starboard/build/config/BUILDCONFIG.gn

```diff
@@ -313,6 +313,90 @@ template("copy") {
   }
 }
 
+template("evergreen_loader") {
```

Comment:
This should go in its own GNI file

Reply:
Tried and couldn't, because the moment I move this template to its own file, the shared_library target in this template will reference the template defined in BUILDCONFIG.gn instead of the native "shared_library" target that just produces a dl.